### DISCUSSION TITLE
This cool project works on Android.

### DISCUSSION THREAD

**Discussion by keven-ma** [dlemstra/Magick.NET#1481] [General] [answered]
Thanks for making this great project.
I'm using unity3d to develop a vr app for oculus quest. I want to display images from user's local storage. I compiled Magick.Native using Clang in ndk. The entire compilation process went quite smoothly and I only made a small modification to Magick.Native/src/ImageMagick/libraries/cairo/util/malloc-stats.c to fix a compile error. 
<img width="437" alt="test" src="https://github.com/dlemstra/Magick.NET/assets/60804914/15c7b646-487c-42ac-b7aa-98c119536648">

Additionally, the rsvg library failed to compile, but since its license is GNU GPLv2, I decided to skip this library.

Overall, Magick.NET runs very well on the Oculus Quest.
Thanks again.

![100618](https://github.com/dlemstra/Magick.NET/assets/60804914/9fa371d1-497b-4caf-ad49-e5ca4d83daa1)

**Comment by dlemstra** [comment, marked as answer]
Thanks for coming here to tell me that 👍🥂

**Comment by ruroman** [comment]
Hello!
I'm interested in using Magick.NET on Andorid. Could you explain me, how is it possible, please?
I want to use MAUI Visual Studio and will be grateful for any information
